core/controllers/collection_editor.py

```diff
@@ -102,20 +104,40 @@ def get(self, collection_id):
         collection = collection_services.get_collection_by_id(
             collection_id, strict=False)
 
+        can_edit = (
+            bool(self.user_id) and
+            self.username not in config_domain.BANNED_USERNAMES.value and
```

Comment:
This check should actually be for whether the username is in the collection editor whitelist or not (there's a config property for this).